### DISCUSSION TITLE
Windows documentation is wrong

### DISCUSSION THREAD

**Discussion by elieobeid7** [helix-editor/helix#9841] [General] [answered]
### Summary

The [documentation](https://docs.helix-editor.com/themes.html) for Windows is wrong. 

It says to place the themes folder and the config folder in `%AppData%\helix\`

First of all, there's no such thing as a `%AppData%` folder, at least not if you're using PowerShell, I don't know what terminal emulator you're using, and secondly, if you do place the theme in `~AppData\helix\` nothing happens because that's the wrong folder

The correct place for them to be located is `~AppData/local/helix` because that's where `helix.log` is located and that's where neovim stores its data

Except, that doesn't work either if I place the config file in there and the theme files in there, neither the theme file nor the themes are applied.

### Platform

Windows 11

### Terminal Emulator

Powershell

### Installation Method

winget

### Helix Version

helix 23.10 (f6021dd0)

**Comment by kirawi** [comment]
`AppData` is an env variable (so terminal emulator doesn't matter) that resolves to `~/AppData/roaming/` IIRC. It needs to be encapsulated by `%` because that's the syntax to access environment variables (may have been changed with powershell, not sure).

https://learn.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables#variables-that-are-recognized-only-in-the-user-context

**Comment by elieobeid7** [comment]
@kirawi that doesn't work, windows 11, latest version of powershell

<img width="612" alt="image" src="https://github.com/helix-editor/helix/assets/660142/201f5aec-486e-44da-8906-203328242240">

**Reply by kirawi** [reply, marked as answer]
The syntax on Powershell is indeed different, try `cd $env:APPDATA`

**Reply by elieobeid7** [reply]
@kirawi perfect that works thank you so much
